src/util/publishMode.native.ts

```diff
@@ -0,0 +1,2 @@
+/** Returns the publish mode query string. */
+export const publishMode = () => true
```

Comment:
You'll want to hardcode this to `false` on native.

Reply:
Done